phpBB/phpbb/db/migration/data/v330/storage.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v330;
+
+class storage extends \phpbb\db\migration\migration
+{
+	public function update_data()
+	{
+		return array(
+			array('config.add', array('storage_attachment', 'phpbb\\storage\\provider\\local')),
+			array('config.add', array('storage_avatar', 'phpbb\\storage\\provider\\local')),
+			array('config.add', array('storage_backup', 'phpbb\\storage\\provider\\local')),
+			array('config.add', array('avatar/path', $this->config['avatar_path'])),
```

Comment:
forget what I said in the sql file, the key should be: `storage\<storage>\config\<config_option>`